### ISSUE TITLE
Enable PrestaShop users to choose to either use memcached for backoffice, frontoffice, or both

### ISSUE THREAD

**Issue by go-sudo** [PrestaShop/PrestaShop#13389]
#### Is your feature request related to a problem?
When I activate memcached on my website some modules ares stopped like newsletter( mails still in progress and cant' send them) so I dont want cache my backend only frontend

#### Describe the solution you'd like
Enable PrestaShop users to choose to either use memcached for backoffice, frontoffice, or both

---
Hello I need you help to diable the memcached for the backend office please

**Comment by khouloudbelguith** [comment]
Hi @go-sudo,

To disable the Memcached: go to BO => Configure => Advanced Parameters => Performance.

Thanks for check and feedback.

**Comment by go-sudo** [comment]
Hi
What I meant to say is that I want to disable the MEMCACHED to cache the backend of website.

**Comment by khouloudbelguith** [comment]
@go-sudo, thank you to tell me what is your exact problem?

**Comment by go-sudo** [comment]
When I activate memcaced on my website some modules ares stopped like newsletter( mails still in progress and cant' send them)  so I dont want cache my backend only frontend 
i found this on forum but I dont know if that works 
https://www.prestashop.com/forums/topic/335988-backoffice-pages-dont-refresh-after-any-modification/?tab=comments#comment-1883775

**Comment by khouloudbelguith** [comment]
Hi @go-sudo,

It seems that your issue is not a PrestaShop's core bug but most likely a customization problem.
You should check your host.
I close the ticket, feel free to open a new one when needed.

**Comment by go-sudo** [comment]
Hi

I think you should add this on feature because it worth nothing to cache the backend (back office) of the website
Regards

**Comment by khouloudbelguith** [comment]
@marionf, @colinegin, what do you think?

**Comment by colinegin** [comment]
@PrestaShop/prestashop-core-developers what do you think ?

**Comment by matks** [comment]
> Hi
> 
> I think you should add this on feature because it worth nothing to cache the backend (back office) of the website
> Regards

Hi @go-sudo, although indeed the frontend benefits a lot more from cache capabilities than the backend, being able to use memcached for backend is still a useful feature.

So can we rephrase your feature request as "Enable PrestaShop users to choose to either use memcached for backoffice, frontoffice, or both" ?

**Comment by MatShir** [comment]
Hi @go-sudo,
What do you think of matks comment?

**Comment by go-sudo** [comment]
Hello

I think it's good idea to give user that choice to cache the 
Frontoffice
Backoffice
Both
Regards

**Comment by PierreRambaud** [comment]
Yes, we need to be able to specify which cache we want for FO and/or BO

**Comment by go-sudo** [comment]
I want to cache only frontoffice please

**Comment by go-sudo** [comment]
Hello
I need yo cache frontoffice only 
Thanks

**Comment by go-sudo** [comment]
Hello any news please

**Comment by PierreRambaud** [comment]
Still not had the time to implement this feature. If you know how to do it, don't hesitate to help us :)

**Comment by pablomartin89** [comment]
Still with problems with Memcached on backoffice and API/webservice,  returns old values in some cases. 

We need cache only on FrontOffice, how can we do that?

**Comment by PierreRambaud** [comment]
@pmartin-webimpacto Not implemented yet, as far as I know. Ping @khouloudbelguith if you know how to implement Memcached only for frontend?

**Comment by ioweb-gr** [comment]
Hi, 
let me notice that on prestashop 1.7.5.2 this is a huge issue with caching on the backend. I'm trying to change product SEO URLs and with memcached enable although the product saves the new URL, cache is not invalidated. Thus the product URL stays the same as before. 

So my use case is this: 

- Enable memcahed
- Edit a product in backend
- Change the SEO url
- Save the product
- In the JSON Response I can see the URL indeed changed 
- Refresh the page
- The old URL appears instead of the new one. 

So basically the real bug is that cache is not invalidated after certain actions. It's not a terrible idea to cache the backend but you need proper invalidation mechanisms. Otherwise it's best not to cache the backend or the user would undo his changes. 

Now the second bug. 

The clear cache button in performance options doesnt' seem to correctly flush memcache cache. In the same example as before I cleared cache multiple times but nothing happened. 

I disabled the cache option for memcache and immediately I saw my changes reflect in the backend. 

I think this issue needs real attention as it'snot a feature request but a real bug affecting administrators.

**Comment by go-sudo** [comment]
Hello @khouloudbelguith 

I enabled the Nginx cache on Plesk and I can't access to Backoffice, 

https://www.youtube.com/watch?v=aUyERrz9KH8

Regards
